lib/sass/calcite-web/components/_action-bar.scss

```diff
@@ -13,20 +14,30 @@
 
 @mixin action-bar-left() {
   @include clearfix();
+
   .action-bar-btn {
     float: left;
     margin-left: 0;
     margin-right: $baseline * .5;
+
     html[dir="rtl"] & {
       float: right;
-      margin-right: 0;
       margin-left: $baseline * .5;
+      margin-right: 0;
     }
   }
 }
 
 @if $include-action-bar == true {
-  .action-bar {@include action-bar();}
-    .action-bar-btn {@include action-bar-btn();}
-    .action-bar-left {@include action-bar-left();}
+  .action-bar {
```

Comment:
Not sure that newlines make this better. They will only ever have that one mixin declared, so I prefer to keep it tight.